tests/Collection/DropEncryptedCollectionFunctionalTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace MongoDB\Tests\Collection;
+
+use MongoDB\BSON\Binary;
+use MongoDB\Database;
+use MongoDB\Driver\ClientEncryption;
+use MongoDB\Driver\WriteConcern;
+
+use function iterator_count;
+use function str_repeat;
+
+class DropEncryptedCollectionFunctionalTest extends FunctionalTestCase
+{
+    protected ClientEncryption $clientEncryption;
+    protected Database $database;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->skipIfClientSideEncryptionIsNotSupported();
+
+        if ($this->isStandalone()) {
+            $this->markTestSkipped('Queryable encryption requires replica sets');
+        }
+
+        $this->skipIfServerVersion('<', '7.0.0', 'Queryable encryption requires MongoDB 7.0 or later');
+
+        $client = static::createTestClient();
+
+        // Ensure the key vault collection is dropped before each test
+        $collection = $client->selectCollection('keyvault', 'datakeys', ['writeConcern' => new WriteConcern(WriteConcern::MAJORITY)]);
```

Comment:
You'll get used to the new name of this method.
```suggestion
        $collection = $client->getCollection('keyvault', 'datakeys', ['writeConcern' => new WriteConcern(WriteConcern::MAJORITY)]);
```

Why do you need to set the `writeConcern`, as that doesn't guarantee that all replica are up to date.

Reply:
I'm borrowing that from the CSFLE spec tests, which use `w:majority` for all ops targeting the key vault collection. I reckon it makes little difference for tests, but in a production environment this would ensure that any modifications will persist through a replica set election.